static/src/stylesheets/_common.scss

```diff
@@ -21,4 +21,5 @@
 @import 'module/crosswords/_main';
 @import 'module/email/_main';
 @import 'module/_accessibility';
+@import 'module/_badging';
```

Comment:
By moving `_badging` out of `head.facia.scss` and `head.content.scss` the css for resizing the images is now only done on second paint, so there is a moment when the page first loads where the badges are huge. Can we move this back ?